tree/ntuple/v7/test/ntuple_show.cxx

```diff
@@ -84,44 +84,46 @@ TEST(RNTupleShow, BasicTypes)
    std::ostringstream os;
    ntuple2->Show(0, os);
    // clang-format off
-   std::string fString{ std::string("")
-      + "{\n"
-      + "  \"pt\": 5,\n"
-      + "  \"db\": 9.99,\n"
-      + "  \"byte\": 0x89,\n"
-      + "  \"int\": -4,\n"
-      + "  \"uint\": 3,\n"
-      + "  \"uint64\": 44444444444,\n"
-      + "  \"string\": \"TestString\",\n"
-      + "  \"boolean\": true,\n"
-      + "  \"uint8\": 97,\n"
-      + "  \"int8\": 97,\n"
-      + "  \"bitset\": \"10000000000000000000000000000010000000000000000000000000000010010\",\n"
-      + "  \"pstring\": \"abc\",\n"
-      + "  \"atomic\": false\n"
-      + "}\n" };
+   std::string fString{
+R"({
```

Comment:
This PR both this syntax and compile time string concatenation (https://github.com/root-project/root/pull/17784#discussion_r1963744923).

The main disadvantage (not fatal) is that this section is not indented like the surrounding code (whereas the other syntax is).

Reply:
True, but I'd argue the advantage in having a "heredoc-style" string where you see the expected output 1:1 outweighs the misindentation

Reply:
At the time, I must have missed the escaped quotes in the original ....